### PR TITLE
improve menu

### PR DESCRIPTION
**Pull request by hboutemy**

### CODE REVIEW

**Review by slawekjaranowski — APPROVED** [review]

File: src/site/apt/index.apt.vm [hunk 1]
```diff
@@ -26,14 +26,14 @@
 ${project.name}
 
   The Toolchains Plugins enables sharing configuration across plugins,
-  for example to make sure the plugins like compiler, surefire, javadoc, webstart etc. all use the same JDK for execution.
+  for example to make sure the plugins like compiler, surefire, javadoc, webstart etc. all use the same JDK for execution independantly from JRE used to run Maven itself.
```
Comment (pzygielo) [inline src/site/apt/index.apt.vm:29]: If there is place for adverb, it could be `independ*e*ntly` I think.

File: src/site/apt/index.apt.vm [hunk 2]
```diff
@@ -26,14 +26,14 @@
...
   Similarly to the maven-enforcer-plugin, it allows you to control environmental constraints in the build.
 
 * Goals Overview
 
   The Toolchains plugin has one goal:
 
-  * {{{./toolchain-mojo.html}toolchains:toolchain}} checks that toolchains requirements are met by currently configured toolchains.
+  * {{{./toolchain-mojo.html}toolchains:toolchain}} selects a toolchain based on configured requirements and store it in build context for later retrieval by other plugins.
```
Comment (pzygielo) [inline src/site/apt/index.apt.vm:36]: If [it] `select*s*` then it `store*s` I guess.

File: src/site/apt/index.apt.vm (hunk 1 above)
Comment (hboutemy) [reply src/site/apt/index.apt.vm]: fixed, thank you

File: src/site/apt/index.apt.vm (hunk 2 above)
Comment (hboutemy) [reply src/site/apt/index.apt.vm]: fixed, thank you

**Pull request merged**